docs/qt.md

```diff
@@ -64,23 +107,35 @@ It is possible to build OpenDDS without using the configure script, although
 this is not recommended unless one is familiar with OpenDDS and it's build
 system, MPC. This list explains how the configure script tells MPC how to
 build Qt5 applications, but not how to build OpenDDS without the configure
-script in general.
+script in general. These environment variables can also be used with the
+configure script to override their values.
 
-  - The `qt5` feature must be enabled (e.g. `-features qt5=1`).
+  - The `qt5` feature must be enabled (e.g. `perl mpc.pl ... -features qt5=1 ...`).
   - `QTDIR` environment variable points to the root of the Qt instance.
     - This is supplied to the configure script using the `--qt` option and
```

Comment:
This entire section is about configuring the build manually (without the configure script) so the references to the configure script here could be moved elsewhere.